include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -294,4 +310,12 @@
 #    define _ONEDPL_SYCL_INTEL_COMPILER 1
 #endif
 
+// Intel(R) oneAPI DPC++ Compiler cannot build some cases of destroy and destroy_n with unseq policies on Windows
+// It might be fixed in the future releases, which follow 20240000
```

Comment:
probably dont want this comment anymore

Reply:
I left a TODO comment as it was suggested earlier.